.vscode/launch.json

```diff
@@ -19,7 +19,13 @@
       "request": "launch",
       "name": "Jest Current File",
       "program": "${workspaceFolder}/node_modules/.bin/jest",
-      "args": ["${file}", "--config", "jest.config.js", "--no-cache"],
+      "args": [
+        "${file}",
+        "--config",
+        "jest.config.js",
+        "--no-cache",
+        "--testTimeout=2147483647"
```

Comment:
makes it easier to debug tests in VS Code since Jest won't kill the process after the default timeout when we are just looking around while being paused in the debugger